third-party/llvm/find-llvm-config.sh

```diff
@@ -14,6 +15,15 @@ then
 elif command_exists llvm-config
 then
   command -v llvm-config
+elif command_exists /usr/local/opt/llvm@$PREFERRED_VERSION/bin/llvm-config
```

Comment:
are these MacPorts or Homebrew paths or something? Might be worth a comment saying what they're intending to support.